.github/workflows/build-natives.yaml

```diff
@@ -40,18 +40,24 @@ jobs:
           restore-keys: ${{ runner.os }}-m2
       - name: Maven Version Info
         run: mvn -version
-      - if: matrix.os != 'windows-latest'
+      - if: matrix.os == 'macos-latest'
         name: OpenSSL Library Version Info
-        run: openssl version -v
+        run: |
```

Comment:
@fjuma We can remove this step if you prefer, as the same version check is done below on line 58. I left this step here because it is easier to find in the resulting job. And I left the version check on line 58 because it happens in the same step as the actual install

Reply:
Actually I reverted this change, this step shows what is the default version installed on the system, not the version that is used to build openssl natives. Nevermind this comment.